app/views/static_pages/home.html.erb

```diff
@@ -7,3 +7,22 @@
   <%= t :p_2 %>
   <%= t :p_3 %>
 </p>
+
+<% if logged_in? %>
+   <div class="row">
+      <aside class="col-md-4">
+         <section class="user_info">
+            <%= render 'shared/user_info' %>
+         </section>
+         <section class="micropost_form">
+            <%= render 'shared/micropost_form' %>
+         </section>
+      </aside>
+
+      <div class="col-md-8">
+         <h3>Micropost Feed</h3>
+         <%= render "shared/feed" %>
+      </div>
+   </div>
+<% else %>
```

Comment:
you seem missing some codes in else condition, please check the tutorial

Reply:
i just checked in slide, but i cannot find which part i had skipped.